### ISSUE TITLE
created/updated times for an asset (girder item) are also not clear

### ISSUE THREAD

**Issue by yarikoptic** [dandi/dandi-publish#108]
```
$> curl --silent -X GET "http://publish.dandiarchive.org/api/dandisets/000027/versions/0.200721.2222/assets/?page_size=2" -H "accept: application/json" -H "X-CSRFToken: otOJPBXcQzTzzPs29GRQ5PvIui81aQXcZhha4QbKCW3SNX0mKKU2ctaKsjEsLhBi" | jq .
{
...
  "results": [
    {
      "version": {
        "dandiset": {
          "identifier": "000027",
          "created": "2020-07-21T22:22:14.732729Z",
          "updated": "2020-07-21T22:22:14.732762Z"
        },
        "version": "0.200721.2222",
        "created": "2020-07-21T22:22:15.396171Z",
        "updated": "2020-07-21T22:22:15.396295Z",
        "count": 1
      },
...
      "created": "2020-07-21T22:22:16.882594Z",
      "updated": "2020-07-21T22:22:16.882641Z",
      "metadata": {
...
        "uploaded_mtime": "2020-07-21T17:31:55.283394-04:00",
        "uploaded_datetime": "2020-07-21T18:00:36.703727-04:00",
      }
    }
  ]
}
```

what are those `updated` and `created` fields for an asset?  you can see in the metadata the `mtime` which I believe I had tried to set on that girder item when I uploaded it.  I think this is what is known to girder:
```
{'size': 18792, 'ctime': '2020-07-21T22:00:36.362000+00:00', 'mtime': '2020-07-21T17:31:55.283394-04:00'}
```
and mtime matches.  I would expect the API return "updated" to correspond to `mtime` (I keep forgetting that girder's `ctime` is not "creation time" but "change time").  I believe client has no ability to change `ctime` and that is why I was setting mtime.  If `updated` to be mapped to `ctime`, then API should probably provide an additional time for `mtime`.

ATM, client will be doomed to use  `metadata.uploaded_mtime` (where available; and fall back may be to `updated`, at least to get some sensible `mtime` set on the file)

Feel free to "map" to #33, #32 (which lack any detail so I could not tell if they are only about dandiset level or what)

**Comment by yarikoptic** [comment]
or may be I will "harmonize" into having "modified" within client at least, and expect it to appear later or returned by API (so I will not populate from `metadata.uploaded_mtime` if already set, but I will compare and issue a warning may be)

**Comment by brianhelba** [comment]
Fixed by #142. See https://github.com/dandi/dandi-publish/issues/107#issuecomment-684041133 for more details.
